inc/RestCommand.php

```diff
@@ -111,12 +124,19 @@ public function generate_items( $args, $assoc_args ) {
 	public function delete_item( $args, $assoc_args ) {
 		list( $status, $body ) = $this->do_request( 'DELETE', $this->get_filled_route( $args ), $assoc_args );
 		if ( Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
-			WP_CLI::line( $body['id'] );
+			if ( $status < 400 ) {
+				// handles cases where user forget to put a value for --force=<value>
+				WP_CLI::line( isset( $body['previous'] ) ? $body['previous']['id'] : $body['id'] );
+			}
 		} else {
-			if ( empty( $assoc_args['force'] ) ) {
-				WP_CLI::success( "Trashed {$this->name} {$body['id']}." );
+			if ( $status < 400 ) {
+				if ( isset( $body['previous'] ) ) {
+					WP_CLI::success( "Deleted {$this->name} {$body['previous']['id']}." );
+				} else {
+					WP_CLI::success( "Trashed {$this->name} {$body['id']}." );
+				}
 			} else {
-				WP_CLI::success( "Deleted {$this->name} {$body['id']}." );
+				WP_CLI::error( 'Could not complete request.' );
```

Comment:
error codes now set in exit code and/or error text. Put error code at end of line without trailing period to make it easier for parsing.